### DISCUSSION TITLE
为什么alist用电脑可以正常看视频，用手机和平板就播放失败

### DISCUSSION THREAD

**Discussion by crowhidden** [AlistGo/alist#3403] [Q&A] [answered]
一直重新连接然后加载失败

**Comment by amakerlife** [comment, marked as answer]
帮助文档说过了

https://alist.nn.ci/faq/why.html#why-can-t-i-preview-the-video-or-audio-file

**Reply by crowhidden** [reply]
> 帮助文档说过了
> 
> https://alist.nn.ci/faq/why.html#why-can-t-i-preview-the-video-or-audio-file

好的谢谢你，我刚开始用的Safari和edge，没想到移动端的edge也不行，夸克浏览器成功了
